docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -1745,13 +1774,17 @@ The decoding algorithm that produces the uncompressed data is as follows:
                read block count using HTREE_BLEN_D and set BLEN_D
             decrement BLEN_D
             compute context ID, CIDD from CLEN
-            read distance code using HTREED[CMAPD[4 * BTYPE_D + CIDD]]
-            compute distance by distance short code substitution
-         move backwards distance bytes in the uncompressed data and
-            copy CLEN bytes from this position to the uncompressed
-            stream, or look up the static dictionary word, transform
-            the word as directed, and copy the result to the
-            uncompressed stream
+            read distance code using HTREED[CMAPD[4*BTYPE_D + CIDD]]
+            if distance code is not zero,
```

Comment:
It seems you accidentally removed a line from here:

"compute distance by distance short code substitution"